de_web_plugin.cpp

```diff
@@ -77,6 +77,7 @@ const quint64 macPrefixMask       = 0xffffff0000000000ULL;
 const quint64 legrandMacPrefix    = 0x0004740000000000ULL;
 const quint64 dishMacPrefix       = 0x0008890000000000ULL;
 const quint64 ikeaMacPrefix       = 0x000b570000000000ULL;
+const quint64 danalockMacPrefix   = 0x000b570000000000ULL; // note: same as ikeaMacPrefix
```

Comment:
Should we just keep one and rename it to `silabs1MacPrefix`?

https://maclookup.app/search/result?mac=000b570

Reply:
You overlooked the other duplicate entry ;)

I'd prefer to leave it here as is and have a seperate PR for that as we need to propagate the changes through all files.

Reply:
Sounds good